### ISSUE TITLE
LeakyReLU?

### ISSUE THREAD

**Issue by BrunoVox** [twtygqyy/pytorch-SRResNet#25]
Regarding your SRResNet implementation (the generator), I was wondering why you used a LeakyReLU instead of a PReLU with a trainable "a". Reading the paper and seing your implementation raised this question, because the two are related, but a bit different. It seems to me that the author meant a trainable PReLU in the original implementation.

Also, why did you use InstanceNorm2d instead of BatchNorm2d? Were you feeding the network with single images instead of a 16 sized batch as the paper specifies?

My last question would be what dataset you used to train your implementation? I've seen you used data augmentation, but I didn't find info on the used dataset.

I'm studying the paper and planning on running it on Imagenet, as the author did, expecting to get similar results. I also will definetly take a look on your evaluation code implementation, the Matlab tip, which increases PSNR values, seems valuable.

My implementation is a bit different than yours and it's getting 29,75dB PSNR on Set5. To get these results I trained it with a sampled 5500 images from Imagenet and I'm hoping to get an improvement when I train it with the 350k images they mention in the paper.
